src/c++/perf_analyzer/test_inference_profiler.cc

```diff
@@ -225,6 +225,22 @@ TEST_CASE("test_check_window_for_stability")
     lp.stability_threshold = 0.1;
     CHECK(TestInferenceProfiler::TestCheckWindowForStability(ls, lp) == true);
   }
+  SUBCASE("test stability window not 5")
+  {
+    ls.infer_per_sec = {500.0, 520.0, 510.0, 505.0, 515.0};
```

Comment:
to expose a potential bug, you likely want something that does stabilize with 3, but does not stabilize with 5. That way it would actually catch an issue if we weren't using the window size correctly.

Reply:
Updated the test